### ISSUE TITLE
Critical bug fixes

### ISSUE THREAD

**Issue by fanlbsg** [Hydreon/Steadfast2#584]
error the spawn in file level.php

**Comment by HerryYT** [comment]
shut up x2.. what this should mean? don't open those useless issues, **you doesn't help the software doing this!** 

> STOP

**Comment by dktapps** [comment]
@HerryYT you're spamming 60 people's notifications with your useless comments, please shut up yourself.

**Comment by HerryYT** [comment]
> @HerryYT you're spamming 60 people's notifications with your useless comments, please shut up yourself.

Sorry, I'm @HerryYT of Telegram, but that does not matter. In any case it is useful what I am doing, if you have also received the notification I am displeased .. I had not against you .. but Steadfast is overloaded with useless issues to close, as you can see there are issues without even the description and with a title "hello" .. this is a problem because it hides the real problems and Hydreon loses time with these people who have nothing to do .. I understand that can be annoying but it is useful.

**Comment by dktapps** [comment]
no, it's not useful at all... when was the last time you saw any of the developers interact with this issue tracker? Hydreon isn't losing any time on this issue tracker because they don't use it. You are simply contributing more unnecessary spam.

**Comment by HerryYT** [comment]
@dktapps It depends, because there is still @vchibikov that sometimes reads some issues and helps, but certainly as you say these do not read them, and you're right ..: D
